roles/fixer/tasks/main.yml

```diff
@@ -10,18 +18,27 @@
   become: yes
 
 - name: get qemu virt_type
-  ansible.builtin.shell: crudini --get /var/lib/config-data/puppet-generated/nova_libvirt/etc/nova/nova.conf libvirt virt_type
+  ansible.builtin.shell: "crudini --get {{ nova_config_path }} libvirt virt_type"
   become: yes
-  register: virt_type
+  register: virt_type_check
+  ignore_errors: true
+
+- set_fact:
+    virt_type="{{ virt_type_check.stdout }}"
+  when: virt_type_check is success
+
+- set_fact:
+    virt_type="kvm"
+  when: virt_type_check is failed
 
 - name: set qemu virt_type
-  ansible.builtin.shell: crudini --set /var/lib/config-data/puppet-generated/nova_libvirt/etc/nova/nova.conf libvirt virt_type qemu
```

Comment:
@cubeek do you remember?

Reply:
A little. I remember VMs going to paused state after they were spawned on the environment used for the workshop. I created a customized jenkins job that provided unified experience for attendees. Instead of configuring it through Infrared and OOO, I just created this task that sets the right virt_type.

I don't think it hurts having here unless it causes any problems. It's conditional and won't touch the env if qemu is set.